server/src/main/java/com/linecorp/centraldogma/server/internal/pluggable/PluggableServices.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.server.internal.pluggable;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.function.Function;
+
+import com.google.common.collect.ImmutableList.Builder;
+
+import com.linecorp.centraldogma.server.pluggable.PluggableService;
+
+public final class PluggableServices {
```

Comment:
Just curious, did you choose this name because there is `Plugin` already? If so, you can ignore it and name this as `Plugin`.

Reply:
Yes, right! Let me rename it. :-)